parmed/amber/_amberparm.py

```diff
@@ -328,6 +332,22 @@ def from_structure(cls, struct, copy=False):
             # Maybe it just has CHARMM parameters?
             raise TypeError('AmberParm does not support all of the parameters '
                             'defined in the input Structure. Try ChamberParm')
+
+        # Convert all RB torsions to propers and delete the originals.
+        for dihedral in struct.rb_torsions:
+            proper_types = DihedralTypeList.from_rbtorsion(dihedral.type)
+            for proper_type in proper_types:
+                proper = _copy.copy(dihedral)
+                proper.type = proper_type
+                struct.dihedrals.append(proper)
+                struct.dihedral_types.append(proper.type)
+                proper.type.list = struct.dihedral_types
```

Comment:
You can remove this `list` assignment if you add the calls to `claim()` described below